cirq-google/cirq_google/json_test_data/AnalogDetuneQubit.json

```diff
@@ -0,0 +1,10 @@
+{
+    "cirq_type": "AnalogDetuneQubit",
+    "length": 10,
+    "w": 5,
+    "target_freq": 8,
+    "prev_freq": null,
+    "neighbor_coupler_g_dict": null,
```

Comment:
Should we provide values for these parameters in the json test?

Reply:
We should but `The expression must eval correctly when only cirq, pandas as pd, numpy as np and sympy have been imported.`  Maybe it is good to add `tunits` as import dependency in future PR? wdyt?

Reply:
I think it might be worth adding tunits to the import list eventually in the import setup for json, but that can be done later.

Reply:
yes, add a TODO in the `json_serialization_test.py` to mark this taks.